src/drivers/bmi160/bmi160.hpp

```diff
@@ -227,8 +227,10 @@
 #define BMI160_GYRO_DEFAULT_RANGE_DPS		2000
 #define BMI160_ACCEL_DEFAULT_RATE           800
 #define BMI160_ACCEL_MAX_RATE               1600
```

Comment:
That's true; that value does correctly specify the maximum sample rate of the accel per the datasheet, but the ioctl for max rate uses the faster gyro rate to set the hrt callback. I have no strong opinion on removing it; happy to defer.